### ISSUE TITLE
Hyperspace persecution.

### ISSUE THREAD

**Issue by walterar** [pioneerspacesim/pioneer#3664]
Many times I tried to pursue a ship through hyperspace without success. Does anyone has achieved it? Have you ever worked this for someone?

**Comment by impaktor** [comment]
I did some investigation on entry and depature clouds +1 year ago, and I think it worked then.

**Comment by laarmen** [comment]
Hasn't the system entry point been randomized?

**Comment by fluffyfreak** [comment]
If you're following a hyperspace cloud then it shouldn't be random should it? Otherwise why follow the cloud?

**Comment by impaktor** [comment]
Right. They have the same entry point. Or at least used to.

Key here is to check arrival date on the depature cloud. It could be you arrive _days_ prior to the ship you're following. (or days too late).

**Comment by walterar** [comment]
Mmm ...  I still do not know of anyone who has pursued a ship through hyperspace successfully.
I fear that "Hypercloud Analyzer" is, for the moment, a decorative element without practical use. It seems that this feature was never completed.

**Comment by fluffyfreak** [comment]
@walterar actually just above @impaktor said that he tested it last year and it worked. So yes it was completed and it did work.

When using it you might arrive a long time before, or after, the ship you are following because of the way hyperspace works.

To test this I think it would be helpful to have a way of spawning a ship identical to the players, have it hyperspace to another system, target it's hyperspace cloud and follow it.

Since the ships would be identical you should arrive very closely after it. In theory.

**Comment by walterar** [comment]
@fluffyfreak  The only reference is in TradeShips.lua

`ship = Space.SpawnShip (ship_name, 9, 11, {from, dest_time})`

You must be 'dest_time' before, **of course**; but you must **also** be at the point of entry, which is between 9 and 11 light years of the star.

@impaktor  was really very lucky to be there ;)

**Comment by walterar** [comment]
Well, somebody has to do it, I will try to improve this feature. This is closed for now.

**Comment by fluffyfreak** [comment]
We'll keep it open so we can track it and confirm that it's working or not, and that it's been fixed :)

**Comment by walterar** [comment]
Okay. We will offer a reward to anyone who can prove that he can pursue and kill the damn that jumped into hyperspace. :)

**Comment by robothauler** [comment]
I can confirm that this works. At least partially.
I found the ship within 1AU distance. Except the ship jumps into an unpopulated system.

**Comment by robothauler** [comment]
Here are 2 savefiles:
http://hauler.lusc.net/pioneer/Sensor_test_1
http://hauler.lusc.net/pioneer/Sensor_test_2

The Sinonatrix ET-9975 will take off at 04:20:30

**Comment by fluffyfreak** [comment]
Excellent, thanks @robothauler

**Comment by walterar** [comment]
@robothauler Yes. That's handled TradeShips.lua and that does not work in uninhabited systems. In Assessination.lua I not see nothing in the code that makes it possible.
